WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/InsightsItem.kt

```diff
@@ -2,6 +2,8 @@ package org.wordpress.android.ui.stats.refresh
 
 abstract class InsightsItem(val type: Type) {
```

Comment:
Should this be a sealed class (with `ListInsightItem` as a data class within it)?

Reply:
The issue I have with sealed classes is that you always have to have all the children within the same file. In this case it totally makes sense though 👍